examples/one-2-one/src/repositories/supplier.repository.ts

```diff
@@ -0,0 +1,31 @@
+import {Getter, inject} from '@loopback/core';
+import {
+  DefaultCrudRepository,
+  juggler,
+  repository,
+  HasOneRepositoryFactory,
+} from '@loopback/repository';
+import {Supplier, Account} from '../models';
+import {AccountRepository} from './account.repository';
+
+export class SupplierRepository extends DefaultCrudRepository<
+  Supplier,
+  typeof Supplier.prototype.id
+> {
+  public readonly accounts: HasOneRepositoryFactory<
```

Comment:
Since the relation is 1-1, each supplier has only one account. Let's rename this property to `account` please.